install/pre-installation.md

```diff
@@ -642,6 +642,12 @@ Follow the [Prepare Site Init](prepare_site_init.md) procedure.
 
 1. (`pit#`) Setup boot links to the artifacts extracted from the CSM tarball.
 
+   1. Patch the `set-sqfs-links.sh` script to include the blacklisting of an undesired kernel module.
+
+        ```bash
+        sed -i -E 's:rd.luks=0 /:rd.luks=0 module_blacklist=rpcrdma \/:g' /root/bin/set-sqfs-links.sh
```

Comment:
![image](https://user-images.githubusercontent.com/7772179/183999073-842e21e6-d14a-443d-97f4-6eff3035cf22.png)

This looks kind of weird.

Can this be in a blockquote instead of a nested, ordered list of one?